docs/main/eventing-communication/README.md

```diff
@@ -1,21 +1,48 @@
-# Using Knative Kafka broker
-If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+# Using Knative broker for eventing communication
 
-Kafka perfectly fullfills this reliability need.
+## Automated installation steps
+Usage:
+```
+Usage: ORCHESTRATOR_NAME=ORCHESTRATOR_NAME BROKER_NAME=BROKER_NAME BROKER_NAMESPACE=BROKER_NAMESPACE [KAFKA_REPLICATION_FACTOR=KAFKA_REPLICATION_FACTOR] [ORCHESTRATOR_NAMESPACE=openshift-operators] [BROKER_TYPE=kafka] [INSTALL_KAFKA_CLUSTER=true] ./eventing-automate-install.sh
+  ORCHESTRATOR_NAME                   Name of the installed orchestrator CR
+  ORCHESTRATOR_NAMESPACE              Optional, namespace in which the orchestrator operator is depoyed. Default is openshift-operators
+  BROKER_NAME                         Name of the broker to install
+  BROKER_NAMESPACE                    Namespace in which the broker must be installed
+  BROKER_TYPE                         Optional , type of the broker. Either 'kafka' or 'in-memory'. Default is: 'kafka'
+  INSTALL_KAFKA_CLUSTER               Optional, if set to true, indicates that kafka cluster must be installed. Will only be used if BROKER_TYPE is 'kafka'. Default is: true
```

Comment:
pls replace everywhere that makes sense `kafka` with `Kafka`

Reply:
done